src/packages/README.md

```diff
@@ -30,33 +103,67 @@ the [CheriBSD ports issue tracker](https://github.com/CTSRD-CHERI/cheribsd-ports
 
 **CheriABI packages** are compiled using pure-capability CHERI C/C++, and
 employ fine-grained C/C++ memory protection.
-There are currently over 8,000 CheriABI packages available.
-These packages are considered *experimental*.
 
-These packages are installed in the standard `/usr/local` hierarchy.
+There are currently over 8,000 CheriABI packages available, including:
+* Development utilities: `git`;
+* Editors: `nano`;
+* Networking software: `rsync`;
+* Shells: `dash`.
+
+These packages are considered *appropriate for experimental use*.
+
+The packages are installed in the standard `/usr/local` hierarchy as they match
+the base system ABI.
 
 *Note:* These packages compile, but many have CHERI-related warnings that
-have not been audited and only a limited set have been tested.  Bugs
-related to CHERI support can be reported in the
-[CheriBSD ports issue tracker](https://github.com/CTSRD-CHERI/cheribsd-ports/issues).
+have not been audited and only a limited set have been tested. Bugs related to
+CHERI support can be reported in the [CheriBSD ports issue
+tracker](https://github.com/CTSRD-CHERI/cheribsd-ports/issues).
+While the CheriABI packages are more interesting as they use CHERI memory-safety
+features, you must remember that they might break in run-time and hence might
+not be suitable for critical operations.
+In such case, a corresponding hybrid ABI package might be considered instead of
+a CheriABI package.
```

Comment:
I feel we might be warning too much and not pointing out what they are good for enough. We should point out somewhere (probably before the warnings) that cheriabi packages can provide dependencies for software development and provide a list of example libraries.

Reply:
Done.

Reply:
I've suggested some additions specifying that the point of the packages is earlier in the text.